tests/GenerateSriHashTest.php

```diff
@@ -67,4 +60,14 @@ public function it_can_hash_using_sha512()
 
         $this->assertEquals("sha512-{$base64Hash}", Sri::hash('files/app.css'));
     }
+
+    /** @test */
+    public function it_returns_an_empty_string_in_disabled_environments()
```

Comment:
Can you change the name for something more descriptive of what the test does (return `''` when disabled)?